fineract-provider/build.gradle

```diff
@@ -21,11 +21,11 @@ Run as:
 gradle clean tomcatrunwar
 '''
 
-project.ext.springBootVersion = '2.1.9.RELEASE'
-project.ext.springVersion = '5.1.10.RELEASE'
-project.ext.springOauthVersion = '2.3.6.RELEASE'
-project.ext.jerseyVersion = '1.17'
-project.ext.springDataJpaVersion = '2.2.0.RELEASE' // also change spring-boot-gradle-plugin version below
+// project.ext.springBootVersion = '2.1.9.RELEASE'
+// project.ext.springVersion = '5.1.10.RELEASE'
+// project.ext.springOauthVersion = '2.3.6.RELEASE'
+// project.ext.jerseyVersion = '1.17'
```

Comment:
Given that this Jersey version is repeated twice, how about keeping this single declaration, and using it in the 2 places below?